plugins/techdocs-node/api-report.md

```diff
@@ -226,9 +233,23 @@ export type ReadinessResponse = {
 // @public
 export type RemoteProtocol = 'url' | 'dir';
 
+// @public
+export type ShouldBuildParameters = {
```

Comment:
open question for @backstage/techdocs-maintainers - how would you feel about skipping this type entirely, and instead just inlining it as `shouldBuild(options: { entity: Entity })`? :) Arguably just adds a bit of noise here to have as a named type.

Reply:
Only issue I see here would be that this would result in a breaking change since this was a publicly exported type. I could skip moving the export to this package (and inlining it in `shouldBuild()`) and leave it in `techdocs-backend` with a deprecation tag but there is a minor risk that the type def between the inlined and named types becomes out of sync due to a future change before the named type is fully deprecated

Reply:
The deprecation isn't breaking, those who imported it from the old package can continue doing so during the deprecation period.

Yeah feature drift is always a risk during deprecations. Even your direct re-exports are dangerous since breakages spread silently across packages (you can't see the changes  in the api report of the re-exporting package).

Reply:
good point - updated to not reintroduce the `ShouldBuildParameters` type